tests/fp8/test_fp8_parameter.py

```diff
@@ -0,0 +1,25 @@
+import torch
+from nanotron.fp8.constants import DTypes
+from nanotron.fp8.meta import FP8Meta
+from nanotron.fp8.parameter import FP8Parameter
+from nanotron.fp8.tensor import FP8Tensor
+
+
+# @pytest.mark.parametrize("quantize_input", [True, False])
+def test_create_fp8_parameter():
+    tensor = torch.randn(16, 16, device="cuda", dtype=torch.float32)
+    # tensor = FP8Tensor(tensor, DTypes.FP8E4M3) if quantize_input else tensor
+
+    fp8_parameter = FP8Parameter(tensor, DTypes.FP8E4M3)
+
+    # TODO(xrsrke): are there better ways for testing
+    # if a tensor is quantized or not?
+    assert isinstance(fp8_parameter.data, FP8Tensor)
+    assert fp8_parameter.requires_grad is True
+    assert fp8_parameter.grad is None
+    assert isinstance(fp8_parameter.fp8_meta, FP8Meta)
+    assert isinstance(fp8_parameter.data.fp8_meta, FP8Meta)
+
+
+# TODO(xrsrke): add test for preventing torch autograd do the backward pass
```

Comment:
Yes good test as well, to make sure bwd functions as expected

Reply:
will do it in another PR 🥲